README.md

```diff
@@ -139,20 +148,24 @@ console.log(data);
 
 ### Utils
 
+A number of utility functions can be found under `utils`, which check whether a document is a valid OA file that has been wrapped or signed. The `diagnose` function can also be used for troubleshooting. 
+
 ```js
 import { utils } from "@govtechsg/open-attestation";
 utils.isWrappedV3Document(document);
 ```
+You can replace `isWrappedV3Document` in the above example with the following values:
+
+- `isWrappedV2Document`: This is the type guard for the wrapped V2 document.
+- `isSignedWrappedV2Document`: This is the type guard for the signed V2 document.
+- `isSignedWrappedV3Document`: This is the type guard for the signed V3 document.
+- `isWrappedV3Document`: This is the type guard for the wrapped V3 document.
 
-- `isWrappedV2Document` type guard for wrapped v2 document
-- `isSignedWrappedV2Document` type guard for signed v2 document
-- `isSignedWrappedV3Document` type guard for signed v3 document
-- `isWrappedV3Document` type guard for wrapped v3 document
-- `diagnose` tool to find out why a document is not a valid open attestation file (wrapped or signed document)
+In addition, the `diagnose` tool can find out why a document is not a valid OpenAttestation file (wrapped or signed document).
 
 ### Obfuscating data
 
-`obfuscateDocument` removes a key-value pair from the document's data section, without causing the file hash to change. This can be used to generate a new document containing a subset of the original data, yet allow the recipient to proof the provenance of the document.
+The `obfuscateDocument` function removes a key-value pair from the document's data section, without causing the file hash to change. You can use this to generate a new document containing a subset of the original data, without letting the recipient prove the provenance of the document.
```

Comment:
"without letting the recipient prove the provenance of the document" is wrong, the recipient will still be able to prove it

Reply:
deleted this line